hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHA.java

```diff
@@ -470,4 +481,12 @@ protected void createKeyTest(boolean checkSuccess) throws Exception {
     }
   }
 
+  protected void waitForLeaderToBeReady()
```

Comment:
nit: should we add it to `MiniOzoneHAClusterImpl` so that it could be used by others?

Reply:
Good point.  Let's move it and clean up related code in a followup.

Reply:
Created HDDS-9524.